express-zod-api/src/documentation-helpers.ts

```diff
@@ -423,7 +411,7 @@ const fixReferences = (
         if (depiction) {
           entry.$ref = ctx.makeRef(
             depiction.id || depiction, // avoiding serialization, because changing $ref
```

Comment:
this `.id` was removed. need to check how it works now. possibly add `$id` alternative

Reply:
added test for it